run_service.sh

```diff
@@ -177,11 +177,7 @@ get_private_key() {
 }
 
 # Function to warm start the policy
-warm_start() {
-    echo '["claude-prediction-offline", "claude-prediction-online", "deepmind-optimization", "deepmind-optimization-strong", "prediction-offline", "prediction-offline-sme", "prediction-online", "prediction-online-sme"]' | sudo tee "$PWD/../$store/available_tools_store.json"  > /dev/null
-    echo '{"counts": [23, 16, 54, 52, 20, 113, 33, 54], "eps": 0.1, "rewards": [0.21330030417382723, -0.06394516434480157, 0.5042296458897277, 0.38925697131774417, -0.2978133327512751, 1.055336834629253, -0.5935249657470777, 0.507192767958923]}' | sudo tee "$PWD/../$store/policy_store.json"  > /dev/null
-    echo '{}' | sudo tee "$PWD/../$store/utilized_tools.json"  > /dev/null
-}
+
```

Comment:
We can either leave this function as is or delete these lines as well:

https://github.com/valory-xyz/trader-quickstart/blob/35e4963e34b9d8771de8ceeacebccb0f61adaf7a/run_service.sh#L816-L817

Reply:
Added warm start back